test/profile.jl

```diff
@@ -11,6 +11,17 @@ end
 
 Profile.clear()
 @profile busywait(1, 20)
+
+let r = Profile.retrieve()
+    mktemp() do path, io
+        serialize(io, r)
+        close(io)
+        open(path) do io
+            @test isa(deserialize(io), Tuple{Vector{UInt64},Dict{UInt64,Vector{StackFrame}}})
```

Comment:
when backporting I think these `UInt64`s may need to be changed to `UInt` for this to pass on 32-bit

[github decided to hide this, reposting it]